tests/foreach/foreach.t

```diff
@@ -0,0 +1,95 @@
+SETUP PYTHON
+
+$ [ -n "$PYTHON" ] || PYTHON="`which python`"
+$ [ -n "$PYTHONPATH" ] || PYTHONPATH="$TESTDIR/.." && export PYTHONPATH
+
+TEST FOREACH WITH COMMAND
+
+  $ cp -r $TESTDIR/test_dir_1 test_dir_1
```

Comment:
As said in chat, this directory is currently a broken submodule - git submodule crashes with 

    fatal: no submodule mapping found in .gitmodules for path 'tests/foreach/test_dir_1'

so I cannot understand exactly what is going on here. And in general, again as anticipated, I would avoid submodules completely, even if just for testing, as the quark repository is meant to be standalone (esp. due to internal mirrors).

Reply:
Changed in https://github.com/comelz/quark/pull/27/commits/4c7534ce4606f97c571aa39e4458c2c800b95e7b